.eslintignore

```diff
@@ -1,2 +1,3 @@
 build
-node_modules
\ No newline at end of file
+node_modules
+.eslintrc.json
```

Comment:
I figure eslint is formatted and has a really strict schema so we could ignore it all together.